build.gradle.kts

```diff
@@ -82,13 +67,16 @@ allOpen {
 }
 
 java {
-  toolchain.languageVersion.set(JavaLanguageVersion.of(19))
+  toolchain.languageVersion.set(JavaLanguageVersion.of(21))
 }
 
 tasks {
   withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
-    kotlinOptions {
-      jvmTarget = "19"
+    compilerOptions {
+      apiVersion.set(org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_2_1)
+      freeCompilerArgs.set(listOf("-Xemit-jvm-type-annotations"))
```

Comment:
out of interest what did you need all of this for?  We normally just do https://github.com/ministryofjustice/hmpps-prisoner-location-api/blob/main/build.gradle.kts#L39-L49

Reply:
fixed